source/javascripts/components/unified-editor/WorkflowConfig/components/PipelineConditionsCard.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect } from 'react';
+import { Meta, StoryObj } from '@storybook/react';
+import BitriseYmlProvider from '@/contexts/BitriseYmlProvider';
+import { MockYml } from '@/core/models/BitriseYml.mocks';
+import { usePipelinesPageStore } from '../../../../pages/PipelinesPage/PipelinesPage.store';
+import PipelineConditionsCard from './PipelineConditionsCard';
+
+export default {
+  component: PipelineConditionsCard,
+  decorators: [
+    (Story) => {
+      const { setPipelineId, setWorkflowId } = usePipelinesPageStore();
+
+      useEffect(() => {
+        setPipelineId('pipeline-1');
+        setWorkflowId('workflow-1');
+      }, [setPipelineId, setWorkflowId]);
```

Comment:
Not sure what's the best way to mock the store in the story, without useEffect, the browser tab froze.